cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/AbstractCgmesTapChangerBuilder.java

```diff
@@ -52,19 +51,7 @@ protected TapChanger build() {
         addSteps();
         int neutralStep = p.asInt(CgmesNames.NEUTRAL_STEP);
         int normalStep = p.asInt(CgmesNames.NORMAL_STEP, neutralStep);
-        int position = AbstractObjectConversion.fromContinuous(p.asDouble(CgmesNames.STEP, p.asDouble(CgmesNames.SV_TAP_STEP, normalStep)));
-        if (position > highStep || position < lowStep) {
-            position = neutralStep;
-        }
-        Integer solvedPosition = null;
-        double solvedPositionFromSv = p.asDouble(CgmesNames.SV_TAP_STEP);
-        if (!Double.isNaN(solvedPositionFromSv)) {
-            solvedPosition = AbstractObjectConversion.fromContinuous(solvedPositionFromSv);
-            if (solvedPosition > highStep || solvedPosition < lowStep) {
-                solvedPosition = position;
-            }
-        }
-        tapChanger.setLowTapPosition(lowStep).setTapPosition(position).setSolvedTapPosition(solvedPosition);
+        tapChanger.setLowTapPosition(lowStep).setTapPosition(normalStep).setSolvedTapPosition(normalStep);
```

Comment:
I think this isn't correct to always initialize the solved tap position.
The IIDM TapChanger interface perfectly authorizes null solved tap position:
```java
    /**
     * Get the solved tap position.
     * Can be null if no calculation has been done on the network.
     * <p>
     * Depends on the working variant.
     * @see VariantManager
     * </p>
     */
    Integer getSolvedTapPosition();
```

My opinion is that we should only set the solved tap position if we have it in the input CGMES data, that is if we have imported a SV file. This is already correctly done so in updatePhaseTapChanger from AbstractTransformerConversion:
```java
findValidSolvedTapPosition(ptc, phaseTapChangerId, context).ifPresent(ptc::setSolvedTapPosition);
```
The only thing to change is to remove the initial setting:
```suggestion
        tapChanger.setLowTapPosition(lowStep).setTapPosition(normalStep);
```